File: 251011/정상적인 구구단/normal-multiple.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a;
    cin >> a;

    for(int i = 1; i <= a; i++) {
        for(int j = 1; j <= a; j++) {
            if(j == a) {
                cout << i << " * " << j << " = " << i*j;
                break;
            }
            cout << i << " * " << j << " = " << i*j << ", ";
        }
        cout << "\n";
    }
    return 0;
}

File: 251111/정상적인 구구단/normal-multiple.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a = int.Parse(Console.ReadLine());
        for(int i = 1; i <= a; i++) {
            for(int j = 1; j <= a; j++) {
                if(j == a) {
                    Console.Write($"{i} * {j} = {i*j}");
                } else {
                    Console.Write($"{i} * {j} = {i*j}, ");
                }
            }
            Console.WriteLine("");
        }
    }
}
